torch/_library/utils.py

```diff
@@ -37,3 +37,15 @@ def get_source(stacklevel: int) -> str:
     frame = inspect.getframeinfo(sys._getframe(stacklevel))
     source = f"{frame.filename}:{frame.lineno}"
     return source
+
+
+def parse_namespace(name: str) -> Tuple[str, str]:
+    splits = name.split("::")
+    if len(splits) != 2:
+        raise ValueError(
+            f"Expected `name` to be of the form "
```

Comment:
would be nice to name this `qualified_name` just to avoid the slight name = namespace::name confusion, but thats probably too bc-breaking

Reply:
It's not BC-breaking. We haven't included impl_abstract in a release yet. I can update all of these to qualified_name for less confusion.

Reply:
Going to do `qualname` for short unless we want to be verbose